assests/javaScript/home.js

```diff
@@ -0,0 +1,39 @@
+const movieList = document.getElementById("movie-list");
+let markUp = "";
+
+fetch(
+  `https://api.themoviedb.org/3/movie/top_rated?api_key=d2aa1d6a7ef9316cd7e2270dd937b843&language=en-US&page=1`
+)
+  .then((res) => res.json())
+  .then((data) => showMovies(data.results))
+  .catch((error) => console.log(`error: ${error}`));
+
+function showMovies(movies) {
+  movies.forEach((movie) => {
+    markUp = `<li class="movie-box" id="movie-box">
+      <a href="./movieDetail.html?${queryParam(movie)}" }>
```

Comment:
would you please comment a sample of `movie` here so I can check out how data looks like? And also the correct url that needs to be created in the end :relaxed:

Reply:
You don't have to convert all properties to query params I guess. `id` wouldn't be enough?